lib/openstudio-standards/prototypes/ashrae_90_1/ashrae_90_1_2019/ashrae_90_1_2019.Model.rb

```diff
@@ -364,4 +364,50 @@ def model_add_lights_shutoff(model)
 
     return true
   end
+
+  # This function is used to add zone ERV
+  #
+  # This function is only used for nontransient dwelling units (Mid-rise and High-rise Apartment)
+  def model_apply_prototype_hvac_efficiency_adjustments(model)
```

Comment:
I'm a little bit concern about making this function template specific. I don't think that the aim of this function is to be specific to energy recovery. I'd suggest to wrap the current code in the main `model_apply_prototype_hvac_efficiency_adjustments` method, which deals only with heat exchangers mods, in one method and make that method template specific. This way if someone adds code to `model_apply_prototype_hvac_efficiency_adjustments` that unrelated to heat exchangers, they don't have to add it to this function as well (in the main `else`).